app/components/send_button.js

```diff
@@ -1,60 +1,71 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React, {memo} from 'react';
+import React, {PureComponent} from 'react';
 import {View} from 'react-native';
 import PropTypes from 'prop-types';
+import {intlShape} from 'react-intl';
 
 import TouchableWithFeedback from 'app/components/touchable_with_feedback';
+import {accessibilityProps} from 'app/utils/accessibility';
+import {t} from 'app/utils/i18n';
 import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
 
 let PaperPlane = null;
 
-function SendButton(props) {
-    const {theme} = props;
-    const style = getStyleSheet(theme);
-
-    if (!PaperPlane) {
-        PaperPlane = require('app/components/paper_plane').default;
+export default class SendButton extends PureComponent {
```

Comment:
Why are we changing this component from functional to a class ?

Reply:
It's due to how I did pass react-intl via context but with the comment of passing by props, this will probably reverted into function.